### ISSUE TITLE
Fix is_supergroup() and is_subgroup() methods of SymmetryGroup class in pymatgen.symmetry.groups

### ISSUE THREAD

**Issue by blondegeek** [materialsproject/pymatgen#274]
See pymatgen google group discussion: https://groups.google.com/forum/#!topic/pymatgen/iP1tb5OkO7Q

**Comment by shyuep** [comment]
Tess, I just pushed a quick fix that uses data obtained form the bilbao crystallographic server. Pls test it and see if it works. I haven't tried to really implement a true super-sub group matching. I merely saved the tree of relationships and used that for the super and sub group test.

**Comment by blondegeek** [comment]
Hi Shyue Ping, Thanks for adding the fix -- grabbing data from Bilbao was also my first thought as well -- if only they had an API! I'll test what you've added.

**Comment by shyuep** [comment]
Yeah, they don't have an API. But I cheated and basically used a combination of requests url downloading and postprocessing with BeautifulSoup to grab the data. It wasn't that much effort. 

The point group data is missing, but there are only 32 of them. So that;s a smaller problem.

**Comment by blondegeek** [comment]
I've created similar scripts using mechanize and BeautifulSoup. For example, I often need to compare whether two structures are actually distortions of one another. To do this, I created functions that grab data from Bilbao's STRUCTURE RELATIONS app (http://www.cryst.ehu.es/cryst/rel.html) so that I can determine whether it's possible to transform the higher symmetry structure into it's lower symmetry setting.
